server/cmd/mmctl/commands/user.go

```diff
@@ -809,21 +816,22 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 		}
 	}
 
+	params := []string{}
```

Comment:
Should `url.Values` be used instead of a string slice? That type also handles the encoding.

Reply:
Yep. updated